sphinx/writers/latex.py

```diff
@@ -462,12 +465,17 @@ def declare_package(packagename, options=None):
             if tocdepth >= SECNUMDEPTH:
                 # Increase secnumdepth if tocdepth is depther than default SECNUMDEPTH
                 self.elements['secnumdepth'] = '\\setcounter{secnumdepth}{%d}' % tocdepth
-        if getattr(document.settings, 'contentsname', None):
-            self.elements['contentsname'] = \
-                self.babel_renewcommand('\\contentsname', document.settings.contentsname)
         # allow the user to override them all
         self.check_latex_elements()
         self.elements.update(builder.config.latex_elements)
+
+        if self.elements['babel'] != '3.1415':
+            self.elements['multilingual'] = self.elements['babel']
```

Comment:
people who in the past use ``'babel'`` key for ``\\usepackage{polyglossia}`` with xelatex/lualatex will be ok. But if they set the ``'babel'`` to empty and used ``'preamble'`` key for polyglossia, it is no good.

this is complicated: one option would be to say in the documentation that people with xelatex/lualatex want to control exactly the way they use babel or polyglossia **must** use ``'babel'`` key for that.